arborator/main.py

```diff
@@ -454,6 +590,10 @@ def cluster_reporter(config):
             print(f'Min members needs to be an integer {min_members}')
             sys.exit()
 
+    if min_members < 2:
+        message = f'{MINIMUM_MEMBERS_KEY} ({min_members}) needs to be at least 2.'
+        raise Exception(message)
+
     if not force and os.path.isdir(outdir):
         print(f'folder {outdir} already exists, please choose new directory or use --force')
         sys.exit()
```

Comment:
Should be `sys.exit(1)` to exit with non-zero code (or raise an Exception).

Reply:
Updated in https://github.com/phac-nml/arborator/commit/5c79a5ba2b88f91bd5626fd904ed1fe291e0c17e